src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalAmount;
+
+  if (days >= 7) {
+    totalAmount = (days * costPerDay) - 50;
+  } else if (days >= 3) {
+    totalAmount = (days * costPerDay) - 20;
+  } else {
+    totalAmount = days * costPerDay;
+  }
```

Comment:
You can just immediately return the full price with discount, so you won't need to use else...if